skore/pyproject.toml

```diff
@@ -38,7 +38,7 @@ Issues = "https://github.com/probabl-ai/skore/issues"
 
 [project.scripts]
 skore-ui = "skore.__main__:main"
-skore-hub-login = "skore.hub.login:login"
+skore-hub-login = "skore.hub.authentication.login:login"
```

Comment:
It is the time that I bring back from the dead an old discussion: https://github.com/probabl-ai/skore/issues/723#issuecomment-2544009508